db/migrate/20200504210159_remove_site_id_from_nodes.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+class RemoveSiteIdFromNodes < ActiveRecord::Migration[6.0]
+  def up
+    remove_index :alchemy_nodes, :site_id
+    remove_column :alchemy_nodes, :site_id, :integer, null: false
+  end
+
+  def down
+    # This IS, in fact, reversible - but it's cumbersome to do. If someone tells me it benefits them
+    # I will implement.
```

Comment:
I think we do not need to take care of the data, no down migration does that. So using `change` here instead of `up` and `down` and a single `remove_column` should do the trick.

Reply:
The column is marked as `null: false`, so we would, in fact, have to take care of the data, which is why I raise here :\

Reply:
Dang. We should have all the data through language, right?